crates/re_data_store/src/editable_auto_value.rs

```diff
@@ -0,0 +1,43 @@
+/// A value that is either determined automatically by some heuristic, or specified by the user.
```

Comment:
We're adding more and more things to re_data_store that don't seem to be related at all to the data store :thinking: